modules/odo-command-changes.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-migration-guide.adoc
+
+:_content-type: REFERENCE
+[id="odo-command-changes_{context}"]
+
+= Commands added, modified or removed in v3
+
+The following table contains a list of `odo` commands that have been added, modified or removed in v3. An *X* indicates that the command is not present in that version.
+
+[cols="2,2,3",options="header"]
```

Comment:
Great job with this table :) 

I'm really impressed with how well it's documented, considering how gigantic it is!